resourcematch-app/src/pages/index/index.page.tsx

```diff
@@ -1,9 +1,66 @@
+import { ResponseList } from '../../components/ResponseList'
+import { TextInputForm } from '../../components/TextInputForm'
+import { BookSelection, Book } from '../../components/BookSelection'
 export { Page }
+import styled from 'styled-components'
+import { useState } from 'react'
+
+const CenteredContainer = styled.div`
+  display: flex;
+  width:100%;
+  flex-direction: column;
+  align-items: center;
+`
 
 function Page() {
+  const [responses, setResponses] = useState<string[]>([]);
+  const [slug, setSlug] = useState('')
+  const books: Book[] = [{
+    friendlyName: 'Algebra',
+    slug: 'college-algebra-corequisite-support-2e'
+  }, {
+    friendlyName: 'History',
+    slug: 'world-history-volume-2'
+  }
+  ]
+  const handleSubmit = async (input: string): Promise<void> => {
+    setResponses([])
+    const urls: string[] = await callMatchApi(input)
+    setResponses(urls)
+  }
+
+  const callMatchApi = async (input: string) => {
+
+    const response = await fetch('http://localhost:8888/match', {
```

Comment:
This code will always try `localhost:8888`, even when deployed. We'll want to something similar to `env.ts` in `k12-apps-raise` where we set the endpoint based on build. 

Also, let's decouple the endpoint from the route. For example:

```javascript
export const ENV = {
  RESOURCEMATCH_API: import.meta.env.MODE === 'production' ? 'https://resourcematch-api.k12.openstax.org' : 'http://localhost:8888'
}
```

and 

```javascript
const response = await fetch(`${ENV.RESOURCEMATCH_API}/match`, {
```